Homeworks/Homework_2/task.py

```diff
@@ -0,0 +1,119 @@
+import requests as re
```

Comment:
solve pycharm's warnings
![image](https://github.com/MykytaKa/PyCourse/assets/43804299/e6dc5c66-647b-4511-9acc-44af68439327)

Reply:
ask questions if you need explanation of the warnings